uaclient/cli.py

```diff
@@ -91,23 +91,24 @@ def new_f(args, cfg):
     return new_f
 
 
-def require_valid_entitlement_name(operation: str):
-    """Decorator ensuring that args.name is a valid service.
+def require_valid_entitlement_names(operation: str):
+    """Decorator that verifies which of the args.names is a valid
+    service or not.
```

Comment:
It feels like keeping this decorator  is now overkill given that we aren't doing anything other than setting "entitlements_found".

Minimally we aren't using operation param anymore.

I think this would be better served as a standalone function like this get_valid_entitlement_names that enable and disable can call directly.
```suggestion
def get_valid_entitlement_names(                                                
    names: 'List', allow_beta: bool = False                                     
) -> 'List'                                                                     
    """Return a list of valid entitlement names.                                
                                                                                
    :param allow_beta: Consider beta entitlements as valid                      
                                                                                
    :return: List of valid entitlement names                                    
    """                                                                         
    entitlements_found = []                                                     
    for ent_name in names:                                                      
        ent_cls = entitlements.ENTITLEMENT_CLASS_BY_NAME.get(ent_name)          
        if ent_cls:                                                             
            if not ent_cls.is_beta:                                             
                entitlements_found.append(ent_name)                             
            elif allow_beta:                                                    
                entitlements_found.append(ent_name)                             
    return entitlements_found   
```